### PR TITLE
Add login/logout and authentication with Auth0

### PR DESCRIPTION
**Pull request by schneider-clarisse**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules/
+src/js/*.config.js
```
Comment (larajanecka) [inline .gitignore:2]: How sure are we that we don't want to share config files for stuff? I could see this blowing up on our faces in the future.

File: src/js/pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { PropTypes as T } from 'react'
```
Comment (larajanecka) [inline src/js/pages/Login.js]: Can we maybe name proptypes as something more descriptive then T? Maybe expand it to Types, or just leave it as proptypes. If its too long to type we can expand it with `const {object, number, etc} = PropTypes`

File: src/js/util/jwtHelper.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  return date
+}
+
+export function isTokenExpired(token){
+  const date = getTokenExpirationDate(token)
+  const offsetSeconds = 0
+  if (date === null) {
+    return false
+  }
+  return !(date.valueOf() > (new Date().valueOf() + (offsetSeconds * 1000)))
```
Comment (larajanecka) [inline src/js/util/jwtHelper.js]: the Moment library used for events might make this a little bit easier

**Review by larajanecka — APPROVED** [review]
Looks good to me. Couple little styling things and a quick question is all.

File: .gitignore (hunk 1 above)
Comment (schneider-clarisse) [reply .gitignore:2]: I mean, we can fix it specifically later. Or rename. Or move it to a secret folder? I don't quite know convention. Happy to fix it later.

File: src/js/pages/Login.js (hunk 2 above)
Comment (schneider-clarisse) [reply src/js/pages/Login.js]: sure.

File: src/js/util/jwtHelper.js (hunk 3 above)
Comment (schneider-clarisse) [reply src/js/util/jwtHelper.js]: good catch

**Pull request merged**